mulog-kinesis/src/com/brunobonacci/mulog/publishers/aws_utils.clj

```diff
@@ -0,0 +1,25 @@
+(ns com.brunobonacci.mulog.publishers.aws-utils
+  (:require
+    [cognitect.aws.client.api :as aws]
+    [cognitect.aws.credentials :as credentials]
+    [cognitect.aws.util :as u])
+  (:import (java.nio ByteBuffer)))
+
+(def region (u/getenv "AWS_REGION"))
+
+(def kinesis (aws/client {:api                  :kinesis
+                          :region               (keyword region)
+                          :credentials-provider (credentials/environment-credentials-provider)}))
+(defn publish!
+  [stream-name records]
+  (aws/invoke kinesis {:op      :PutRecords
+                       :request {:StreamName stream-name
+                                 :Records    records}
+                       }))
+(defn create-record!
+  [k d]
+  {:PartitionKey k
+   :Data         (-> d
+                     .getBytes
+                     ByteBuffer/wrap)
+   })
```

Comment:
not too sure that we need to turn this into a byte []... 

as far as i remember kinesis should be able to accepts strings, no need to encode in base64

Reply:
thanks, fixed